wradlib/vis.py

```diff
@@ -73,7 +73,7 @@ def __repr__(self):
 
     def fix_cyclic(self):
         rays = self._obj.azimuth
-        dim0 = rays.dims[0]
+        dim0 = rays.dims[0].sort(reverse = False)
```

Comment:
@rexdivakar Thanks for taking a stab at this. This is not quite what it should do. I'll try to explain a bit more:

- `self._obj` needs to be sorted over it's `dim0` before `rays` is extracted
- for xarray Dataset you would need to use `.sortby()` method